httpclient/src/main/kotlin/tbdex/sdk/httpclient/TbdexHttpClient.kt

```diff
@@ -17,12 +19,23 @@ import tbdex.sdk.protocol.models.Offering
 import tbdex.sdk.protocol.serialization.Json
 import tbdex.sdk.protocol.serialization.Json.jsonMapper
 import web5.sdk.dids.Did
+import java.io.File
+import java.net.InetAddress
 
 /**
  * A client of the tbDEX HTTP interface for communicating with a PFI.
  */
 object TbdexHttpClient {
-  private val client = OkHttpClient()
+  private val client = run {
```

Comment:
does changing of this mean that the tests have to change?

i tried running the test locally (got a mock pfi up and running whose service endpoint is localhost:9000) and am getting this error
```
java.net.UnknownHostException: private hosts not resolved
at okhttp3.dnsoverhttps.DnsOverHttps.lookup(DnsOverHttps.kt:69)
at tbdex.sdk.httpclient.TbdexHttpClient.getExchanges(TbdexHttpClient.kt:177)
at tbdex.sdk.httpclient.TbdexHttpClient.getExchanges$default(TbdexHttpClient.kt:159)
```

Reply:
hmm. looks like private host like localhost:9000 is something that the new client does not resolve